client/master.go

```diff
@@ -627,7 +636,7 @@ func (m *masterClient) RemoveNodeMeta(ctx context.Context, nodeID entity.NodeID)
 		return err
 	}
 	masterServer.reset()
-	response, err := m.HTTPPost("/meta/remove_server", string(reqBody))
+	response, err := m.HTTPPost(ctx, "/meta/remove_server", string(reqBody))
```

Comment:
It's not necessary.auto recover ps We need to pay more attention to data recovery and data loading. We have verified that auto recover ps can recover data under the condition of purely using raft logs, and there may still be problems with the way of using snapshots.